docs/includes/file-tutorial-excel-custom-functions.md

```diff
@@ -153,7 +182,7 @@ Complete the following steps to create a custom function named `stockPrice` that
     }
     ```
 
-4. You must reregister the add-in in Excel in order for the new function to be available to end-users. Complete the following steps for the platform that you're using in this tutorial.
+5. You must reregister the add-in in Excel in order for the new function to be available to end-users. Complete the following steps for the platform that you're using in this tutorial.
```

Comment:
Also looks like you would need to run a build after adding the stock price function. when you run 'npm run start' again, it will sideload to office desktop so you don't need to do any additional steps on registration in Excel (it just runs).